src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -391,7 +390,7 @@ export default {
 		 */
 		menuPlacement: {
 			type: String,
-			default: 'bottom',
+			default: 'auto',
```

Comment:
Can we keep this at `bottom`? It works the equally well and does not change the old behaviour. Or was there a specific reason / design decision for this change?
```suggestion
			default: 'bottom',
```